contributed_definitions/nyaml/NXpeak_model.yaml

```diff
@@ -0,0 +1,140 @@
+category: base
+doc: |
+  Peak fitting of a one-dimensional spectrum.
```

Comment:
Why necessarily restrict to one-dimensional?

Also, more generic, description of a fit procedure